search-service/src/main/kotlin/com/egm/stellio/search/model/AttributeInstanceResult.kt

```diff
@@ -0,0 +1,12 @@
+package com.egm.stellio.search.model
+
+import java.net.URI
+import java.time.ZonedDateTime
+
+data class AttributeInstanceResult(
+    val attributeName: String,
+    val instanceId: URI? = null,
```

Comment:
instanceId can't be null, we are generating one for each record

Reply:
When searching attribute instances with a non null timeBucket, we are not returning an instanceId and the result is mapped to AttributeInstanceResult.
For this reason i made the field nullable.

Reply:
Yes, indeed.